### ISSUE TITLE
Cursor jumps during locked drag

### ISSUE THREAD

**Issue by daveriedstra** [daveriedstra/draggy#3]
"However that `-l` lock option doesn't work well. Cursor jumps around when doing drags after the first one.  https://la.wentropy.com/yW-8.webm"

_Originally posted by @amosbird in https://github.com/daveriedstra/draggy/issues/2#issuecomment-491596266_

**Comment by amosbird** [comment]
The screencast is gone but the issue is still reproduceable easily. (on a thinkpad t480 machine at least)

**Comment by daveriedstra** [comment]
> The screencast is gone but the issue is still reproduceable easily. (on a thinkpad t480 machine at least)

Yep, can reproduce on my own machine. The fix is not so easy, however, and unfortunately I don't have a lot of time to work on this project at the moment. I would be happy to review a PR though!
